code/kildishev-maksim/src/main/java/ru/unn/agile/figuresvolumecalculator/model/constants/MathConstants.java

```diff
@@ -0,0 +1,11 @@
+package ru.unn.agile.figuresvolumecalculator.model.constants;
+
+public final class MathConstants {
+
```

Comment:
Better to make the MathConstants "package-private" class. It is needed only inside the package, but now you can use it from the outside.

Reply:
Fixed!